libsoftwarecontainer/component-test/filegateway_componenttest.cpp

```diff
@@ -69,3 +69,43 @@ TEST_F(FileGatewayTest, TestActivateWithMinimalValidConf) {
     ASSERT_TRUE(gw->setConfig(jsonConfig));
     ASSERT_TRUE(gw->activate());
 }
+
+TEST_F(FileGatewayTest, MalformedPath) {
+// Mount a path which is malformed, e.g. too many "/"
```

Comment:
I think the comment should either be indented along with the code inside the test, or located above the test if it's a comment for the whole test. Same goes for some other places below as well.